New version: Circuitscape v5.14.0

JuliaRegistrator opened the pull request:
- Registering package: Circuitscape
- Repository: https://github.com/Circuitscape/Circuitscape.jl
- Created by: @ranjanan
- Version: v5.14.0
- Commit: 50c77b5e8632851690b694739275a1ff74249047
- Reviewed by: @ranjanan
- Reference: https://github.com/Circuitscape/Circuitscape.jl/commit/50c77b5e8632851690b694739275a1ff74249047#commitcomment-145813924
- Description: Algorithms from circuit theory to predict connectivity in heterogeneous landscapes
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd629a0fa3e8c4e3d5f13fb8cdf004e6d9c293510622040baf21d8e22c8625bf23c2fe910ddb4d0edd1535f83eeaa933d2d5d5aaa1757d04aebc8210cf4d8b892327210480151e47178dad378a4d7bb57a05566405c80c7d103c178c7cf51ddc1630041af42041696a73c8f30f0ffedc0e1cb30f4c00f128657d0ef19242fdddfd4d6c7acccbc8ce83f8570eafeec2a7f08e56908265dd49c19fb37bbbd767f2a6bdd1d451e67f8acb5ab80071ddf556239 -->

ViralBShah commented:
This requires Julia 1.11. Is it ok to merge - or should it wait for Julia 1.11 to release?

JuliaRegistrator committed:
New version: Circuitscape v5.14.0

UUID: 2b7a1792-8151-5239-925d-e2b8fdfa3201
Repo: https://github.com/Circuitscape/Circuitscape.jl.git
Tree: 55848c10a213daac77e821104b7ce5eed9b958a1

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

ViralBShah commented:
Waiting on Julia 1.11 release.

ViralBShah committed:
Merge branch 'master' into registrator-circuitscape-2b7a1792-v5.14.0-5565c01803
